UI/MapWnd.cpp

```diff
@@ -4119,24 +4118,26 @@ void MapWnd::InitFieldRenderingBuffers() {
     }
     m_field_scanline_circles.createServerBuffer();
 
-    for (auto& [field_texture, buffers] : m_field_vertices) {
-        if (!field_texture)
-            continue;
-
-        // TODO: why the binding here?
-        glBindTexture(GL_TEXTURE_2D, field_texture->OpenGLId());
-        glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_S, GL_CLAMP);
-        glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_T, GL_CLAMP);
+    std::size_t max_buffer_size = 0;
 
-        buffers.first.createServerBuffer();
-        buffers.second.createServerBuffer();
+    auto field_texture_exists = [](const auto& field_vertices_pair) { return !!field_vertices_pair.first; };
```

Comment:
lambdas that capture nothing can generally be `static constexpr`

Reply:
Cheers, make sense, thank you, will change this